2024-11-09T17:01:13.078Z [ERROR] Eccezione non gestita: lpTracker.start is not a function
2024-11-09T17:01:31.845Z [ERROR] Eccezione non gestita: lpTracker.start is not a function
2024-11-09T17:02:41.305Z [ERROR] Eccezione non gestita: lpTracker.start is not a function
2024-11-09T17:05:24.660Z [ERROR] Eccezione non gestita: lpTracker.start is not a function
2024-11-09T17:25:10.567Z [ERROR] Eccezione non gestita: lpTracker.start is not a function
2024-11-09T17:25:19.808Z [ERROR] Eccezione non gestita: lpTracker.start is not a function
2024-11-09T17:32:36.138Z [ERROR] Errore nella chiamata RPC: fetch failed
2024-11-09T17:32:36.138Z [ERROR] Errore nel recupero del saldo: fetch failed
2024-11-09T17:32:36.138Z [ERROR] Error getting personal wallet info: fetch failed
2024-11-09T17:33:05.206Z [ERROR] Errore nella chiamata RPC: fetch failed
2024-11-09T17:33:05.206Z [ERROR] Errore nel recupero del saldo: fetch failed
2024-11-09T17:33:05.206Z [ERROR] Error getting personal wallet info: fetch failed
2024-11-09T17:33:35.207Z [ERROR] Errore nella chiamata RPC: fetch failed
2024-11-09T17:33:35.207Z [ERROR] Errore nel recupero del saldo: fetch failed
2024-11-09T17:33:35.207Z [ERROR] Error getting personal wallet info: fetch failed
2024-11-09T17:35:47.007Z [ERROR] Eccezione non gestita: listen EADDRINUSE: address already in use :::5001
2024-11-12T08:52:53.697Z [ERROR] Errore nello scraping Coinbase: Traceback (most recent call last):
  File "/Users/manuelmartorana/Projects/Apps/crypto/solana_tracker/backend/coinbaseScraper.py", line 1, in <module>
    from selenium import webdriver
ModuleNotFoundError: No module named 'selenium'

2024-11-12T08:52:53.725Z [ERROR] Errore nello scraping Coinbase: Traceback (most recent call last):
  File "/Users/manuelmartorana/Projects/Apps/crypto/solana_tracker/backend/coinbaseScraper.py", line 1, in <module>
    from selenium import webdriver
ModuleNotFoundError: No module named 'selenium'

2024-11-12T08:53:24.610Z [ERROR] Errore nello scraping Coinbase: Traceback (most recent call last):
  File "/Users/manuelmartorana/Projects/Apps/crypto/solana_tracker/backend/coinbaseScraper.py", line 1, in <module>
    from selenium import webdriver
ModuleNotFoundError: No module named 'selenium'

2024-11-12T08:53:24.633Z [ERROR] Errore nello scraping Coinbase: Traceback (most recent call last):
  File "/Users/manuelmartorana/Projects/Apps/crypto/solana_tracker/backend/coinbaseScraper.py", line 1, in <module>
    from selenium import webdriver
ModuleNotFoundError: No module named 'selenium'

2024-11-12T08:54:44.293Z [ERROR] Errore nello scraping Coinbase: Traceback (most recent call last):
  File "/Users/manuelmartorana/Projects/Apps/crypto/solana_tracker/backend/coinbaseScraper.py", line 1, in <module>
    from selenium import webdriver
ModuleNotFoundError: No module named 'selenium'

2024-11-12T08:54:44.313Z [ERROR] Errore nello scraping Coinbase: Traceback (most recent call last):
  File "/Users/manuelmartorana/Projects/Apps/crypto/solana_tracker/backend/coinbaseScraper.py", line 1, in <module>
    from selenium import webdriver
ModuleNotFoundError: No module named 'selenium'

